orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/job/RunJobStage.groovy

```diff
@@ -34,6 +34,9 @@ class RunJobStage implements StageDefinitionBuilder {
     builder
       .withTask("runJob", RunJobTask)
       .withTask("monitorDeploy", MonitorKatoTask)
-      .withTask("waitOnJobCompletion", WaitOnJobCompletion)
+
+    if (!stage.getContext().containsKey("ignoreResults") || (Boolean.parseBoolean(stage.getContext().get("ignoreResults").toString()) == true)) {
```

Comment:
I'd maybe call it `waitForCompletion` rather than `ignoreResults`.

```
def waitForCompletion = !stage.getContext().getOrDefault("waitForCompletion", "true").equalsIgnoreCase("false")
if (Boolean.valueOf(waitForCompletion)) {
   builder.withTask(...)
}
```

^^ might be a tad tidier.

Feels like it needs decent messaging in the UI but otherwise I think this is alright ... what does @robfletcher think?